rskj-core/src/main/java/org/ethereum/config/Constants.java

```diff
@@ -39,6 +39,7 @@ public class Constants {
     public static final byte TESTNET_CHAIN_ID = (byte) 31;
     public static final byte DEVNET_CHAIN_ID = (byte) 32;
     public static final byte REGTEST_CHAIN_ID = (byte) 33;
+    public static final String TEST_DATA = "0x603d80600c6000396000f3007c01000000000000000000000000000000000000000000000000000000006000350463c6888fa18114602d57005b6007600435028060005260206000f3";
```

Comment:
it looks like this is tests specific field and only used in `EthModuleTest`. I'd rather move it to tests

Reply:
Done, I moved it to the test file.